TROUBLESHOOTING.md

```diff
@@ -1,5 +1,19 @@
 # Troubleshooting
 
+## Cannot find module `./<module name>.js` from `./<module_name>.spec.ts`
+
+In order to support resolving import paths with a `.js` extension, which source file comes from a TypeScript module, one can add a custom resolver:
+
+```sh
+npm install --save-dev jest-ts-webcompat-resolver
+```
+
+And add to the `jest.config.js`:
+
+```
+resolver: 'jest-ts-webcompat-resolver'
+```
+
```

Comment:
Also, there should be a note here that, if one has another resolver, one should extend it to have extra capability of resolving `<module_name>.js` in `ts` file.